packages/drivers/routerlicious-driver/src/test/restWrapper.spec.ts

```diff
@@ -140,6 +165,33 @@ describe("RouterliciousDriverRestWrapper", () => {
 				errorType: RouterliciousErrorTypes.genericNetworkError,
 			});
 		});
+
+		it("retry query param is appended on subsequent api request", async () => {
+			let retryQueryParamTested = false;
+			// Fail first request with retriable error
+			nock(testHost).get(testPath).reply(401);
+			// Second request must contain the query param "retry=1"
+			nock(testHost)
+				.get(/.*/)
+				.query((q) => {
+					assert(q);
+					assert(q.retry === "1");
+					return true;
+				})
+				.reply(429, { retryAfter: 0.1 });
```

Comment:
These tests are only validating internal retries (401 and 429). It looks like this change is supposed to take into account external retries as well (otherwise we could just use an internal retryCount param, not a URL key Map).

IMO we should add a test or 2 to reflect that functionality. For example, 2 subsequent requests to `restwrapper.get` with the same `testUrl`, where the first request receives a 500